SwiftHEXColors.podspec

```
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "SwiftHEXColors"
-  s.version          = "1.1.2"
+  s.version          = "1.2"
```

Could you change the version to `1.2.0`?
```suggestion
  s.version          = "1.2.0"
```